lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2225,10 +2227,8 @@ def add_collection3d(self, col, zs=0, zdir='z'):
             - PatchCollection
         '''
         zvals = np.atleast_1d(zs)
-        if len(zvals) > 0 :
-            zsortval = min(zvals)
-        else :
-            zsortval = 0   # FIXME: Fairly arbitrary. Is there a better value?
+        zsortval = (
+            np.min(zvals) if zvals.size else 0)  # FIXME: arbitrary default
```

Comment:
Slightly better:
~~~
default_z = 0  # FIXME: arbitrary default
zsortval = np.min(zvals) if zvals.size else default_z
~~~

Reply:
went for some middle ground